charts/webhook-dispatcher/Chart.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: v1
```

Comment:
consistently quote strings (as is recommended practice for helm-charts afaik) - OR consistently _not_ quote string literals (both OK with me)